pycbc/inference/distributions.py

```diff
@@ -1219,6 +1253,195 @@ def from_config(cls, cp, section, variable_args):
         return _bounded_from_config(cls, cp, section, variable_args,
             bounds_required=False)
 
+class FromFile(_BoundedDist):
+    """A distribution that reads the values of the parameter(s) from a file,
+    computes the kde to construct the pdf, and draws random variables from it.
+
```

Comment:
If this is reading from a file, you should say somewhere how the file should be organized. Someone would have to look in the code to figure this out. And I think that is inside a seperate function from this class.